src/models/User.ts

```diff
@@ -0,0 +1,21 @@
+import { Schema, model, connect, Model } from 'mongoose';
+
+interface User {
+    name: string; 
+    email: string; 
+    password: string; 
+    registeredAt: Date; 
+    verified: Boolean; 
```

Comment:
```suggestion
    verified: boolean; 
```

For consistency, all primitive types should be lowercase. This is pretty annoying with mongoose schema expecting uppercase, but for everything non-schema related, we should use lowercase.